examples/app/ios/IOSApp-Bridging-Header.h

```diff
@@ -0,0 +1,11 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+
+#ifndef IOSApp_Bridging_Header_h
+#define IOSApp_Bridging_Header_h
+
+#import "todolist-Bridging-Header.h"
```

Comment:
That reminds me we should really do https://github.com/mozilla/uniffi-rs/issues/266.